layouts/partials/breadcrumb.html

```diff
@@ -18,8 +18,11 @@
     {{ template "breadcrumbnav" (dict "p1" .p1.Site.Home "p2" .p2 )  -}}
   {{ end -}}
   {{ $isActive :=  eq .p1 .p2 }}
-  <li class="breadcrumb-item{{ if $isActive }} active{{ end }}"
-      {{- if $isActive }} aria-current="page"{{ end }}>
```

Comment:
@at055612 - why did you remove `aria-current`?

Reply:
Good question.  I can't remember why I took that out.  Maybe because I don't think it is applicable when the breadcrumb is shown on the taxonomy list, however I agree it ought to be there for the page breadcrumbs.  Will take a look.

Reply:
For now I've reverted this change -- see #1068. We would need a bit more logic to remove (all?) the Aria related labels for the taxonomy-list usage.